src/chainparams.cpp

```diff
@@ -166,10 +166,10 @@ class CMainParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 00000000000000000008a89e854d57e5667df88f1cdef6fde2fbca1de5b639ad
-            /* nTime    */ 1626697539,
-            /* nTxCount */ 656509474,
-            /* dTxRate  */ 2.424920418708139,
+            // Data from RPC: getchaintxstats 4320 000000000000000000052d314a259755ca65944e68df6b12a067ea8f1f5a7091
```

Comment:
Does this need to be 4096 blocks? The reason I changed this to `4320` is that it's what `getchaintxstats` returns by default, but it seems we've always explicitly used 4096.

Reply:
I wonder why 28 days (4096) was used instead of the getchaintxstats default of 30 (4320).